### ISSUE TITLE
Is this experiment being continued in v12?

### ISSUE THREAD

**Issue by lupestro** [gcsga/gcsga#280]
I know this is an experiment and an alpha. I also know it was last updated in January. I was excited about it because it looked like it was headed in the right direction. 

Now I’m just interested in finding out what to expect for this one going forward. When something new is happening, it isn’t uncommon for a lot of the work to happen in private until there is something tangible and useful to show, except for a series of experiments to collect feedback on different areas.

* Is there a second generation of the GURPS system in the works? 
* Will its documents be structured to use actors with embedded items in the customary way? 
* Are any new experiments expected from the effort? 
* Should I discard this experiment from my system at this point? Has it “served its purpose?”

**Comment by crnormand** [comment]
Yes, yes, a resounding YES!   Unfortunately,  Reality has intruded in all of the author's lives.   Jeff's and my jobs started requiring us to go back into the office, and this really restricted our free time, and Mikolaj is getting married!  This is a hobby project, so we work on it when we can (which, as of late, hasn't been too much).

To answer your questions, this IS the second generation system.    The GURPS Game Aid (GGA) is the first generation.   It has a static model (which makes it hard to expand) and no character building tools.    It served its purpose, and it works well enough for what it does, but it cannot be enhanced easily... basically, it hit a dead end.

The GCS Game Aid (GCSGA) is built on the GCS character model, and implements the GCS functionality INSIDE Foundry.    Since the character creation logic is inside of Foundry, equipment can now correctly determine its values based on the individual character stats.   This was all Mikolaj's work, and Jeff and I jumped on the bandwagon to help.

I honestly don't know how we are handling equipment.    A fully fleshed out character from GCS may have hundreds of pieces of equipment.   I don't know if that affects the decision.

My biggest focus is to get back into coding and to migrate the OTF (On The Fly) system to the GCSGA.   Once that is started, I will be going through all of the configuration options available in the GGA, to see if they make sense to migrate.

Up to you... but **I know** there will be major efforts on this system in the fall.   Mainly because I am retiring in October 2024.
So if you can bear with us for a few more months, maybe we can surprise you.

I hope that helps,

Chris

**Comment by crnormand** [comment]
Unfortunately, no.   Life has taken a new direction for me... and it is taking my a lot longer to return to coding than I expected.

Also, it seems like we might headed in the direction of incorporating the GCS character modeling portion of the gcsga into the original gga.    Again, no hard timelines... just some discussions.
